graphene/src/intersection.rs

```diff
@@ -1,7 +1,9 @@
 use std::ops::Mul;
 
-use glam::{DAffine2, DVec2};
-use kurbo::{BezPath, Line, PathSeg, Point, Shape};
+use glam::{DAffine2, DMat2, DVec2};
+use kurbo::{BezPath, Line, ParamCurve, ParamCurveExtrema, PathSeg, Point, Rect, Shape};
+
+pub const F64PRECISION: f64 = f64::EPSILON * 1000.0; // for f64 comparisons
```

Comment:
Perhaps should be defined in the consts file (I know this is not possible as consts is in the editor - but perhaps a file for document consts should be added), Also probably describe more what the cont does e.g. `CURVE_INTERSECTION_PRECISION`